Modules/_statisticsmodule.c

```diff
@@ -0,0 +1,120 @@
+/* statistics accelerator C extensor: _statistics module. */
+
+#include "Python.h"
+#include "structmember.h"
+#include "clinic/_statisticsmodule.c.h"
+
+/*[clinic input]
+module _statistics
+
+[clinic start generated code]*/
+/*[clinic end generated code: output=da39a3ee5e6b4b0d input=864a6f59b76123b2]*/
+
+
+static PyMethodDef speedups_methods[] = {
+    _STATISTICS__NORMAL_DIST_INV_CDF_METHODDEF
+    {NULL, NULL, 0, NULL}
+};
+
+/*[clinic input]
+_statistics._normal_dist_inv_cdf -> double
+   p: 'd'
+   mu: 'd'
+   sigma: 'd'
+[clinic start generated code]*/
+
+static double
+_statistics__normal_dist_inv_cdf_impl(PyObject *module, double p, double mu,
+                                      double sigma)
+/*[clinic end generated code: output=02fd19ddaab36602 input=74c459adf13062f9]*/
+{
+    double q, num, den, r, x;
+    q = p - 0.5;
+    if(fabs(q) <= 0.425) {
+        r = 0.180625 - q * q;
+        // Hash sum: 55.8831928806149014439
```

Comment:
I presume this is all correct, but will you double check the hash sums still add up.

Reply:
I've read the paper http://csg.sph.umich.edu/abecasis/gas_power_calculator/algorithm-as-241-the-percentage-points-of-the-normal-distribution.pdf. And it looks like numbers are correct.